coralproject/talk
# [CORL-1281] Prevent viewNewEdges from being garbage collected

## What does this PR do?

Any data in Relay gets garbage collected unless there is an active query or calling 
`Relay` to specifically retain a query.

This PR inserts data queries to `viewNewEdges` to prevent garbage collection and also
upgrades Relay.

## How do I test this PR?

- Creates comments to trigger `View New Comments` in moderation queue.
- Now click on the `decision history`, this triggers a query and also activates the garbage collector.
- Now clicking on `View New Comments` should be working as expected.
